.gitignore

```diff
@@ -49,3 +48,10 @@ src/moduleTemplates/autoCppModule/*
 **/__pycache__/
 venv/
 .venv/
+
+# VSCode
+.devcontainer
+
+# Python packaging
+*.egg-info
+build/
```

Comment:
@dpad , is it possible to have these two directories be inside `dist3` instead of the root BSK directory?  This way, if someone uses the `--clean` build option old wheel related items are also erased.  Further, it keeps the root BSK folder cleaner?  Thanks for your thoughts on this.  Before I ask my researchers to run this branch, I'd love to have this resolved so they are running a true final candidate that is also rebased on latest develop.